RealmBrowser/Classes/RLMRealmBrowserWindowController.m

```diff
@@ -101,6 +101,20 @@ - (IBAction)saveObjcModels:(id)sender
     [RLMModelExporter saveModelsForSchemas:objectSchemas inLanguage:RLMModelExporterLanguageObjectiveC];
 }
 
+- (IBAction)saveCopy:(id)sender
+{
+    NSString *fileName = [self.modelDocument.presentedRealm.realm.path lastPathComponent];
+    NSSavePanel *panel = [NSSavePanel savePanel];
+    [panel setNameFieldStringValue:fileName];
+    [panel beginSheetModalForWindow:self.window completionHandler:^(NSInteger result){
+        if (result == NSFileHandlingPanelOKButton)
+        {
+            NSURL *fileURL = [panel URL];
+            [self.modelDocument.presentedRealm.realm writeCopyToPath:fileURL.path error:nil];
```

Comment:
This should handle errors, and ideally also run `optimize()` on the tables (see [`-[RLMRealm compact]`](https://github.com/realm/realm-cocoa/blob/v0.94.1/Realm/RLMRealm.mm#L634-L674) for how to do this).